app/scripts/modules/core/src/managed/ManagedWriter.ts

```diff
@@ -3,16 +3,30 @@ import { IPromise } from 'angular';
 import { API } from 'core/api';
 
 export class ManagedWriter {
-  public static pauseResourceManagement(application: string): IPromise<void> {
+  public static pauseApplicationManagement(applicationName: string): IPromise<void> {
     return API.one('managed')
-      .one('application', application)
+      .one('application', applicationName)
       .one('pause')
       .post();
   }
 
-  public static resumeResourceManagement(application: string): IPromise<void> {
+  public static resumeApplicationManagement(applicationName: string): IPromise<void> {
```

Comment:
Do you think there is value to adding tests for these new API methods?

Reply:
We've typically avoided tests around methods if they just make an API call. 

Some of the `*Writer`s do have some tests! But they're on methods that do some processing or branching before making the API call.